### ISSUE TITLE
Securable content types not added to roles permissions

### ISSUE THREAD

**Issue by Skrypt** [OrchardCMS/OrchardCore#2570]
In 01 we could set permissions on content types per user roles by going to each roles and setting content types permissions when you we're setting the content type as "securable".

**Comment by remesq** [comment]
I believe this is in the backlog as it was mentioned by @sebastienros (I think it was on Gitter) that currently, even though you can set Metadata to "Securable", you cannot set permissions on the Content Item.

I'm curious to know, though, if there is a quick/easy way to programmatically encapsulate code to show only items related to the user (whether you attach a part to Content Types with their User info, or whether you do it based on making them the owner of a Content Item). Any thoughts? Sort of how you might do it with roles:

        if (!await _authorizationService.AuthorizeAsync(User, Permissions.RoleName))
        {
            return Unauthorized();
        }

**Comment by sebastienros** [comment]
To remove any confusion I think this issue is about not being able to set permissions at the content __type__ level, in the Roles screen. When we talk about content __item__ permission it's in the content editor, with permission per content item. I am surprises to see content type permissions not implemented, i thought I had done it already. We should ASAP. Content items permissions is less of an issue IMO, but we'll have to do it as an optional feature, it's been popular in O1.

**Comment by Skrypt** [comment]
Yes this is concerning having the ability to set permissions per content types on each user roles. Right now if I create a content type through the admin UI and I set it as "securable"; the permissions checkboxes group won't show in the user roles.

**Comment by remesq** [comment]
My mistake. I saw that problem as well.

I saw your #1091 along with the branch https://github.com/OrchardCMS/OrchardCore/tree/content-permissions. Are there plans to go forward with that, even as an option (i.e., enable the module if wanted)?

**Comment by Skrypt** [comment]
Right now the branch is up for grab. I've not had time to invest in it since we moved forward with other things more important for Beta 3.

**Comment by sebastienros** [comment]
I keep the other one for content items. This one here should be done before though, more important (and simpler).

**Comment by inhumator** [comment]
Could you explain how to use this feature ? 
The content type permissions do show up in role permissions screen but I can't set permissions to just some content-types.

Lets say I have to content types Public and Secured. I want Admin to edit both types but I want Editor to be able to create and edit only the Public type. 
Could you explain what combination of permissions allows me to grant full access to "Public" type and no access to "Secured" type ?

I enabled "Edit Public for others" and enabled all other permisions under "Public:".
I disabled "Edit Secured for others" and disabled all other permissions under "Secured".
Now if I enable "Edit content for others" - editor can create and edit all types both Public and Secured and if I disable "Edit content for others" editor cant create nor edit anything.

Regards.

**Comment by Skrypt** [comment]
Please open a separate issue labelled or named **documentation : securable content types**.
